ui/app/components/sidebar/user-menu.hbs

```diff
@@ -57,7 +57,7 @@
                 @onError={{(fn
                   (set-flash-message "Clipboard copy failed. The Clipboard API requires a secure context." "danger")
                 )}}
-                data-test-copy-button={{this.auth.currentToken}}
+                data-test-copy-button
```

Comment:
This is probably the most impactful change for writing tests. Before you could do something like `[data-test-copy-button].getAttribute('data-test-copy-button'); to return the value being passed to the clipboard. 

With this change we're going to use a sinon spy instead. I suspect this is better practice anyways as we're watching the browsers clipboard which more closely follows real life.

```
// Sinon spy for clipboard
    const clipboardSpy = sinon.stub(navigator.clipboard, 'writeText').resolves();
    await click(GENERAL.copyButton);
    const clipboardText = clipboardSpy.firstCall.args[0]; // you can also use clipboardSpy.secondCall, or by index clipboardSpy.index(9).args[0]
```